src/languages/livescript.js

```diff
@@ -14,7 +14,7 @@ function(hljs) {
       'in if for while finally new do return else break catch instanceof throw try this ' +
       'switch continue typeof delete debugger case default function var with ' +
       // LiveScript keywords
-      'then unless until loop of by when and or is isnt not it that otherwise from to til fallthrough super ' +
+      'then unless until loop of by when and or is isnt not it that otherwise|10 from to til fallthrough super ' +
```

Comment:
10?  If any number would accomplish the same for less I'd use it instead of 10.  "otherwise" is also a plain english word and you're going to make this auto-trigger for comments in other languages that mention the word otherwise... such as :

```cpp
// below should be clear, otherwise see the docs
```

Reply:
Yeah fair enough - I hadn't considered that scenario, just thought it was an unusual keyword.
The value of 10 came from the language author docs:

> The default value for relevance is 1. When setting an explicit value it’s recommended to use either 10 or 0.

Reply:
Those docs should be updated. :-)